src/charm.py

```diff
@@ -123,6 +185,14 @@ def _on_ingress_revoked(self, event: IngressPerAppRevokedEvent) -> None:
         self._update_pebble_layer(event)
         self._update_login_ui_endpoint_relation_data(event)
 
+    @property
+    def _log_level(self) -> str:
+        return self.config["log_level"]
+
+    @property
+    def _tracing_enabled(self) -> bool:
+        return not self.config["tracing_enabled"] == 0
+    
```

Comment:
I feel like it would be a bit more intuitive to have the "tracing_enabled" config value be boolean instead of integer. It's not a big deal though.

Reply:
issue with boolean is how it gets parsed, on this unfortunately i'm out of my depth

if with int i have 0 as `False` and everything else as `True`, if i cast to boolean i dunno what values would be deemed right or wrong, more than happy to change to boolean if you know that

Reply:
I agree with @bencekov, from local tests all of these values work for `false`: `f`, `0`, `false`, `False`, `FALSE` and they are all converted to `false`. (in python you would get a boolean `False` value).

If you provide an incorrect value, juju will apply it and it will return an error.

Reply:
fixed